src/PIL/Image.py

```diff
@@ -24,6 +24,22 @@
 # See the README file for information on usage and redistribution.
 #
 
+if False:
```

Comment:
Is this `if False:` going to remain? Does it need to?

A comment to explain why would help.

Reply:
it should be this:

```python
import typing

if typing.TYPE_CHECKING:
    ...
```